.scripts/run_osx_build.sh

```diff
@@ -48,9 +48,9 @@ set -e
 echo -e "\n\nMaking the build clobber file and running the build."
 make_build_number ./ ./recipe ./.ci_support/${CONFIG}.yaml
 conda build ./recipe -m ./.ci_support/${CONFIG}.yaml --clobber-file ./.ci_support/clobber_${CONFIG}.yaml
-validate_recipe_outputs "lalframe"
+validate_recipe_outputs "lalframe-feedstock"
 
 if [[ "${UPLOAD_PACKAGES}" != "False" ]]; then
   echo -e "\n\nUploading the packages."
-  upload_package --validate --feedstock-name="lalframe" ./ ./recipe ./.ci_support/${CONFIG}.yaml
+  upload_package --validate --feedstock-name="lalframe-feedstock" ./ ./recipe ./.ci_support/${CONFIG}.yaml
```

Comment:
@duncanmmacleod IDK how this happened, but the upload commands here had the wrong text. I think you rerendered from a clone of the feedstock that was in a directory not named like the parent git directory. This is a bug in smithy for sure, but in any case, this PR should fix your uploads.

Reply:
Sounds correct. I clone all of my feedstocks into sub-directories of a common parent, so this one sits at `~/git/feedstocks/lalframe` on my machine. I'll look at smithy to see where that bug is, and be careful in the mean time.